src/plugins/data_source/server/legacy/configure_legacy_client.ts

```diff
@@ -165,10 +165,10 @@ const callAPI = async (
       return request.then(resolve, reject);
     });
   } catch (err) {
-    if (!options.wrap401Errors || err.statusCode !== 401) {
+    if (!options.wrap401Errors || (err as any).statusCode !== 401) {
```

Comment:
```suggestion
    if (!options.wrap401Errors || (err as unknown).statusCode !== 401) {
```

`unknown` will be preferred

Reply:
`err` is `any` with `"useUnknownInCatchVariables": false`, casting to unknown like this `(err as unknown).statusCode` actually results in TS error.